hadoop-hdfs-project/hadoop-hdfs/src/test/java/io/hops/security/TestHopsworksFsSecurityActions.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hops.security;
+
+import io.hops.common.security.FsSecurityActions;
+import io.hops.common.security.HopsworksFsSecurityActions;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.CommonConfigurationKeys;
+import org.apache.hadoop.hdfs.DFSConfigKeys;
+import org.apache.hadoop.security.ssl.KeyStoreTestUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * It assumes there is already a Project in Hopsworks.
+ * Edit USERNAME below
+ */
+@Ignore
```

Comment:
Given the above comment, I guess this is a test to run with hopsworks, but that would probably deserve more explanation in a comment.

Reply:
Yes, both `TestHopsworksFsSecurityActions` and `TestHopsworksRMAppSecurityActions` require a running Hopsworks. I'll make it more clear.